client/src/main/java/com/paypal/selion/internal/platform/pageyaml/YamlV1Reader.java

```diff
@@ -84,6 +85,11 @@ public YamlV1Reader(String fileName) throws IOException {
         return instanceMap;
     }
 
+    @Override
+    public Map<String, String> getGuiMap(String locale, WebDriverPlatform platform) {
+        return getGuiMap(locale);
+    }
```

Comment:
@sebady from comment in com.paypal.selion.reader.YamlV1Reader(line 56), i figured out mobile platform does not need to be supported in yamV1 and i remember i saw somewhere yamlV1 is deprecated if i am not wrong. does it needs to support mobile platform?